.circleci/config.yml

```diff
@@ -221,7 +221,7 @@ jobs:
           command: |
             cd /home/circleci/src/fmriprep
             if [[ "$( git log --format=oneline -n 1 $CIRCLE_SHA1 | grep -i -E '\[skip[ _]?tests\]' )" != "" ]]; then
-              echo "Skipping pytest job"
+              echo "Skipping doc building job"
```

Comment:
This should actually be changed on line 308, not 224.

Reply:
Thanks for pointing this out! Just changed it.